docs/howto_guides/data_plan.md

```diff
@@ -10,15 +10,56 @@ While a new concept, making your DMP machine readable, increases the likelihood
 
 All of the sections below are laid out in the NMDC DMPTool template. This is a living document and the NMDC team welcomes community feedback on this resource. 
 
-![](../_static/images/howto_guides/data_mgt/data_mgt_list.png)
+**Sample and Data Types and Sources:** This section outlines what kinds of data will be produced throughout the project. 
+
+1. Describe the data set including basic identification information, average size, volume of estimated number of data files produced
+2. What types of data are being generated? 
+3. How are they being generated (tools and instruments)? 
+4. What analysis stages will the data go through? 
+5. Will you be using existing data for any of your findings?
+
+**Data Standards and Formats:** This section defines all variables of interest and communicates that you are aware of and will abide by community best practices whenever possible. 
+
+1. How is your data being processed? 
+2. What are the recognized community standards for your data and which will you follow? 
+3. How will your data adhere to [FAIR Data Principles](https://www.go-fair.org/fair-principles/)?
+4. Who will ensure that the data standards and formats are maintained? 
+5. How will you define and categorize variables of interest that are not part of standard fields? List all variables of interest.
+
+**Roles and Responsibilities:** This section shows how your data management plan will be executed and ensures that your team’s data management responsibilities are clearly defined. 
+
+1. Who is responsible for data storage and access, quality control, documentation, and preservation during the project?
+2. Who is responsible for coordinating the various data once collection is complete? 
+3. What is your estimated budget for data management activities?
+
+**Data Dissemination & Archiving:** This section describes what the final data products will be and how you will protect data, if applicable. 
+
+1. What are the anticipated data products? Include secondary products (publications, presentations etc.)  
+2. When will your data be released? 
+3. Who is the target audience for your data set? 
+4. Is the security of your data important for privacy reasons? If so, how do you intend to protect your data?
+
+**Policies for data sharing, public access, and re-use:** This section communicates that you understand your funders data sharing policies and that you have a plan to ensure public availability. 
```

Comment:
I read this as either "data sharing policies of the funder" or "data sharing policies of the funders." In the first case, I'd suggest changing funders to funder's (singular possessive). In the second case, I'd suggest changing funders to funders' (plural possessive). The "suggested change" snippet below shows the latter.

```suggestion
**Policies for data sharing, public access, and re-use:** This section communicates that you understand your funders' data sharing policies and that you have a plan to ensure public availability. 
```